### ISSUE TITLE
Naomi format issue and formula error

### ISSUE THREAD

**Issue by jaliasd** [pepfar-datim/Data-Pack-Feedback#37]
I have a two-fer for you.

Part I
This is a screenshot of the naomi output for zim
![image](https://user-images.githubusercontent.com/8836685/73723482-0f099300-4721-11ea-809f-c48e513d5137.png)
 
This is what the `epi cascade` tab is looking for
![image](https://user-images.githubusercontent.com/8836685/73724368-1467dd00-4723-11ea-8768-9d8af9109243.png)

So the naomi output for zim has '5-9' and the datapack is looking for "05-09". I checked the naomi output for ZAF and it had "05-09". So I'm not sure what's going but it appears to be a Naomi thing.

Part II
The formulas in `epi cascade` tab cell E14 has the formula `=IFNA(ROUND(INDEX(Spectrum!$P:$P,MATCH(RIGHT($A15,13)&"|"&"="&CHAR(34)&$B15&CHAR(34)&"|"&$C15&"|"&E$8,Spectrum!$T:$T,0)),0),"")` per a convo with Abe in SA, he said that in consultation with Scott the formula should be `=IFNA(ROUND(INDEX(Spectrum!$P:$P,MATCH(RIGHT($A15,13)&"|"&$B15&"|"&$C15&"|"&E$8,Spectrum!$T:$T,0)),0),"")` 
Our version of the datapack is 3.0.1 generated on 2020-01-31 21:17:32. 
If I make the above change in E14, that column populates (except for the 05-09 as documented in part 1). However, none of the other columns populate.

Please advise.

**Comment by abeagedew** [comment]
@jaliasd, yes, you are correct the ZAF Naomi has '05-09', not sure if it is something with the Zim output.
